src/listener/Heading.php

```diff
@@ -27,10 +27,19 @@ public function process(Line $line)
 
     /**
      * {@inheritDoc}
+     * 
+     * @since 1.2.0 Added exception
+     * 
+     * @throws \Exception for unknown heading levels
      */
     public function render(\nadar\quill\Lexer $lexer)
     {
         foreach ($this->picks() as $pick) {
+            if (in_array($pick->heading, [1, 2, 3, 4, 5, 6], $strict=true) === false) {
```

Comment:
Same here: As in array can only contain bool return value i would just:

if (!in_array($pick->heading, $this->headingLevels))

Maybe create a property $headingLevels which contains the heading values, this would even allow people to extend. Therefore disable strict compare in `in_array`.

Reply:
Good ones